unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/CommonHUD/Scripts/Toast.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.Builder
+{
+    public interface IToast
+    {
+        /// <summary>
+        /// Show the text
+        /// </summary>
+        /// <param name="instant"></param>
+        void Show(bool instant);
+        
+        /// <summary>
+        /// Set the text and changes the size of the toast based on the text
+        /// </summary>
+        /// <param name="text"></param>
+        void SetText(string text);
+    }
+    
+    public class Toast : BaseComponentView, IToast
+    {
+        [SerializeField] internal float msToBeActive = 2000;
```

Comment:
maybe add a `[Tooltip("Represented in milliseconds")]` and change the type to `int`